impeller/entity/contents/content_context.cc

```diff
@@ -40,6 +40,7 @@ void ContentContextOptions::ApplyToPipelineDescriptor(
   color0.format = color_attachment_pixel_format;
   color0.alpha_blend_op = BlendOperation::kAdd;
   color0.color_blend_op = BlendOperation::kAdd;
+  color0.write_mask = ColorWriteMaskBits::kAll;
```

Comment:
This is just explicitly specifying the default value of the write mask as initialized in the ColorAttachmentDescriptor. Are there cases where we do modify it mask when we create the pipeline descriptor? Perhaps its better to be explicit.

Reply:
Just being explicit.